stable/concourse/templates/web-deployment.yaml

```diff
@@ -319,6 +494,10 @@ spec:
             {{- end }}
             - name: CONCOURSE_DATADOG_AGENT_PORT
               value: {{ .Values.web.metrics.datadog.agentPort | quote }}
+            {{- if .Values.web.metrics.datadog.prefix }}
+            - name: CONCOURSE_DATADOG_PREFIX
+            - value: {{ .Values.web.metrics.datadog.prefix }}
```

Comment:
There's an extra `-` in front of `value` here, it breaks the chart when you use Datadog

Reply:
Thanks for the catch! Fixed.